zds/tutorialv2/managers.py

```diff
@@ -2,6 +2,7 @@
 
 from django.conf import settings
 from django.db import models
+from datetime import datetime
```

Comment:
il me semble que ça n'a jamais été respecté ça. Mais quand je repasserai
dessus, je le ferai.

2016-07-15 19:43 GMT+02:00 victor felder <notifications@github.com>:

> In zds/tutorialv2/managers.py
> <https://github.com/zestedesavoir/zds-site/pull/3276#discussion_r71012299>
> :
>
> > @@ -2,6 +2,7 @@
> >
> >  from django.conf import settings
> >  from django.db import models
> > +from datetime import datetime
>
> D'abord les imports Python. Puis ceux de Django. Puis ceux de ZdS.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zestedesavoir/zds-site/pull/3276/files/b5e0edf701ffee84f83c42cdb8bb8673cecd8e9e#r71012299>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABc_xT0A7Qz6gHvCUJVOfaGAXCOEAgvGks5qV8bfgaJpZM4G_EBV>
> .
>